consensus/consortium/v2/snapshot.go

```diff
@@ -227,33 +230,41 @@ func (s *Snapshot) apply(headers []*types.Header, chain consensus.ChainHeaderRea
 				return nil, consensus.ErrUnknownAncestor
 			}
 
-			isShillin := chain.Config().IsShillin(checkpointHeader.Number)
-			// Get validator set from headers and use that for new validator set
-			extraData, err := finality.DecodeExtra(checkpointHeader.Extra, isShillin)
-			if err != nil {
-				return nil, err
-			}
+			if checkpointHeader.Number.Cmp(common.Big0) == 0 {
+				snap.Validators = make(map[common.Address]struct{})
+				for _, validator := range consortiumCommon.Validators.GetValidators() {
```

Comment:
I think it's better to use `c.contract.GetValidators` here

Reply:
`c.contract` is not accessible on snapshot, that's why I used `Validators` as global variables